docs/integrations.md

```diff
@@ -1,9 +1,9 @@
 # Integration
 
-Zafira is integrated with multiple 3rd-party tools like Amazon, Slack, Gmail etc. You are able to add new integration in working Zafira without restart of web application. Navigate to **Username > Integrations** in top navigation menu:
+Zafira is integrated with multiple 3rd-party tools like Amazon, Slack, Gmail etc. You are able to add new integration in working Zafira without restart of web application. Navigate to **Integrations** icon in sidebar menu:
```

Comment:
Zafira is integrated with multiple third-party tools such as Amazon, Slack, Gmail, etc. You can add new integration to a working Zafira without having to restart the web application. Navigate to the ** Integrations ** icon in the sidebar menu: